homeassistant/components/zwave_js/helpers.py

```diff
@@ -41,6 +42,24 @@
 )
 
 
+@dataclass
+class ZwaveValueID:
+    """Class to represent a value ID."""
+
+    property_: str | int | None = None
+    command_class: int | None = None
+    endpoint: int | None = None
+    property_key: str | int | None = None
+
+    def __post_init__(self) -> None:
+        """Post initialization check."""
+        # If this returns None, all values are None
+        if (
+            self.property_ or self.command_class or self.endpoint or self.property_key
```

Comment:
Maybe use `dataclasses.fields`?

https://docs.python.org/3/library/dataclasses.html#dataclasses.fields

Reply:
that doesn't provide values so I am using `astuple` instead